### PR TITLE
Custom Check Monitor

### PR DESCRIPTION
**Pull request by ruthnaebeck**
### What does this PR do?
- Update Custom Check Monitor doc

### Motivation
Audit of Monitor section

### Preview link
https://docs-staging.datadoghq.com/ruth/monitors7/monitors/monitor_types/custom_check

### Additional Notes
Do not merge until monitor team reviews

### CODE REVIEW

File: content/en/monitors/monitor_types/custom_check.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Custom check monitor
+title: Custom Check Monitor
 kind: documentation
 description: "Monitor status of arbitrary custom checks"
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
description: "Monitor status of arbitrary custom checks."
```

File: content/en/monitors/monitor_types/custom_check.md [hunk 2]
```diff
@@ -16,28 +16,78 @@ further_reading:
...
-  {{< img src="monitors/monitor_types/custom_check/monitor_options.png" alt="monitor options" responsive="true" style="width:80%;">}}
-  While each check run sends a status of either CRITICAL, WARNING or OK, you can choose at what consecutive conditions to cause a state change and a notification. For example, you might want to know immediately when your check fails and only have it recover if it stays that way. In this case you might choose to notify on 1 critical status, 1 warning status and 4 OK statuses.  
-  You can optionally **notify on no data** after a configurable timeframe. You must choose at least 2 minutes for your timeframe.
+Choose a custom check from the drop-down box.
 
-4. Configure your **notification options**:  
-  Refer to the [Notifications][2] dedicated documentation page for a detailed walkthrough of the common notification options.
+### Pick monitor scope
+
+Select the scope to monitor by choosing host names, tags, or choose `All Monitored Hosts`. If you need to exclude certain hosts, use the second field to list names or tags.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
Select the scope to monitor by choosing hostnames, tags, or choose `All Monitored Hosts`. If you need to exclude certain hosts, use the second field to list names or tags.
```

seems like you're referring to the specific ones you choose in this instance. I will not fight you if you disagree though

File: content/en/monitors/monitor_types/custom_check.md [hunk 3]
```diff
@@ -16,28 +16,78 @@ further_reading:
...
+
+* The include field uses `AND` logic. All listed host names and tags must be present on a host for it to be included.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
* The include field uses `AND` logic. All listed hostnames and tags must be present on a host for it to be included.
```

File: content/en/monitors/monitor_types/custom_check.md [hunk 4]
```diff
@@ -16,28 +16,78 @@ further_reading:
...
+* The exclude field uses `OR` logic. Any host with a listed host name or tag is excluded.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
* The exclude field uses `OR` logic. Any host with a listed hostname or tag is excluded.
```

File: content/en/monitors/monitor_types/custom_check.md [hunk 5]
```diff
@@ -16,28 +16,78 @@ further_reading:
...
+{{< tabs >}}
+{{% tab "Check Alert" %}}
+
+A check alert tracks consecutive statuses submitted per check grouping and compares it to your thresholds.
+
+Set up the check alert:
+
+1. Trigger a separate alert for each `<GROUP>` reporting your check.
+
+    Check grouping is specified either from a list of known groupings or by you. For custom check monitors, the per-check grouping is unknown, so it must be specified by you.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
    Check grouping is specified either from a list of known groupings or by you. For custom check monitors, the per-check grouping is unknown, so you must specify it.
```

File: content/en/monitors/monitor_types/custom_check.md [hunk 6]
```diff
@@ -16,28 +16,78 @@ further_reading:
...
+
+2. Trigger the alert after selected consecutive failures: `<NUMBER>`
+
+    Each check run submits a single status of `OK`, `WARN`, `CRITICAL`, or `UNKNOWN`. Choose how many consecutive runs with the `WARN` and `CRITICAL` status trigger a notification. For example, to be notified immediately when your check fails, trigger the monitor alert on `1` critical status or `1` warning status.
+
+3. Choose `Do not notify` or `Notify` for Unknown status.
+
+4. Resolve the alert after selected consecutive successes: `<NUMBER>`
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
4. Resolve the alert after selected consecutive successes: `<NUMBER>`.
```

File: content/en/monitors/monitor_types/custom_check.md [hunk 7]
```diff
@@ -16,28 +16,78 @@ further_reading:
...
+    Choose how many consecutive runs with the `OK` status resolve the alert. For example, to ensure an issue is fixed, resolve the monitor on `4` OK statuses.
+
+{{% /tab %}}
+{{% tab "Cluster Alert" %}}
+
+A cluster alert calculates the percent of checks in a given status and compares it to your thresholds.
+
+Set up a cluster alert:
+
+1. Decide whether or not to group your checks according to a tag. `Ungrouped` calculates the status percentage across all sources. `Grouped` calculates the status percentage on a per group basis.
```
Comment (apigirl) [inline content/en/monitors/monitor_types/custom_check.md]:
```suggestion
1. Decide whether or not to group your checks according to a tag. `Ungrouped` calculates the status percentage across all sources. `Grouped` calculates the status percentage on a per-group basis.
```

File: content/en/monitors/monitor_types/custom_check.md (hunk 2 above)
Comment (ruthnaebeck) [reply content/en/monitors/monitor_types/custom_check.md]: Checked with Cecilia on this. We are going to say this is referring to a group of hosts and not a particular (singular) host, so host names with a space.

**Pull request merged**